benchmark/inference/inference_benchmark.py

```diff
@@ -93,8 +94,14 @@ def run(args: argparse.ArgumentParser) -> None:
                         model.eval()
 
                         start = default_timer()
-                        model.inference(subgraph_loader, device,
-                                        progress_bar=True)
+                        if args.maybe_with_experimental:
```

Comment:
When would this be False? The default is an empty list.

Reply:
Empty list evaluates as `False`.

Reply:
Oh haha understood, thanks.